src/pages/Profile.jsx

```diff
@@ -89,24 +106,43 @@ const Profile = () => {
 
         <ProfileComponent clientProfile={clientProfile} />
 
-        <Button
-          variant="contained"
-          color="secondary"
-          size="small"
-          aria-label="Add Client Button"
-          startIcon={<AddIcon />}
-          onClick={() => setShowModal(true)}
-        >
-          Add Document
-        </Button>
-        <UploadDocumentModal showModal={showModal} setShowModal={setShowModal} />
-        <DocumentTable />
         {!client && (
           <>
-            <SetAclPermsDocContainerForm />
-            <SetAclPermissionForm />
+            <Container sx={{ display: 'flex', justifyContent: 'center', gap: 2 }}>
+              <Button
+                variant="contained"
+                color="primary"
+                size="small"
+                aria-label="Set Container Permissions Button"
+                startIcon={<SettingsIcon />}
+                onClick={() => setShowAclPermsDocContainerModal(true)}
+              >
+                Container Permissions
+              </Button>
+              <Button
+                variant="contained"
+                color="secondary"
+                size="small"
+                aria-label="Add Client Button"
+                startIcon={<AddIcon />}
+                onClick={() => setShowAddDocModal(true)}
+              >
+                Add Document
+              </Button>
+            </Container>
+            <SetAclPermsDocContainerModal
```

Comment:
Given that these are now modals that depends on the `showModal` prop to render, I don't think they need to be kept inside this rendering condition. Think we can have these moved to where `UploadDocumentsModal` and `DocumentTable` are.